power-bi-app/requirements.md

```diff
@@ -27,10 +27,20 @@ To do so:
 * **Click the checkbox to Allow Azure services and resources to access this server (4);** enabling this will allow access from the Power BI App and is located under Exceptions at the bottom of the screen
 * Click **Save (5)** to store preferences
 
+
```

Comment:
gore u dijelu gdje se na početku članka kaže koji stepovi se moraju completeati ('the following steps need to be completed:'), dodati i 'Enable SQL authentication' ili tako nešto